### ISSUE TITLE
Reduce font size for place=neighbourhood

### ISSUE THREAD

**Issue by pnmrnko** [organicmaps/organicmaps#3417]
I noticed that the labels for `place=suburb` are smaller than for `place=neighbourhood`. I don't know if it's a bug, but probably so:

![8F9725BD-00F8-469C-9C61-9E25831B4F26](https://user-images.githubusercontent.com/58340216/190492501-bbb73e11-9850-4184-bbfe-7e94dc161f2f.jpeg)

**Comment by biodranik** [comment]
OSM [says](https://wiki.openstreetmap.org/wiki/Tag:place%3Dsuburb) "For areas within a suburb, the tag [place](https://wiki.openstreetmap.org/wiki/Key:place)=[neighbourhood](https://wiki.openstreetmap.org/wiki/Tag:place%3Dneighbourhood) will usually be used."

**Comment by jpds** [comment]
It also depends on the zoom level, past a certain point `neighbourhood` isn't shown:

![signal-2022-09-15-223937](https://user-images.githubusercontent.com/29158971/190513894-be4b047b-2804-4911-b19a-109bf117bad8.jpeg)

**Comment by pnmrnko** [comment]
What I mean is that there can be many neighborhoods in suburb, but now the text size for neighborhood is even larger than the `place=town` names. At certain scales it gives the impression that the name of the neighborhood is more important than the name of the suburb. Added some more examples: 

![Untitled (Draft)-3](https://user-images.githubusercontent.com/58340216/190585327-896a6dfd-8baa-47be-933f-7c9757e8ab46.jpg)

![Untitled (Draft)-2](https://user-images.githubusercontent.com/58340216/190585360-d66b7aad-20ee-4486-b1f0-bd3d4b27239f.jpg)

**Comment by biodranik** [comment]
Font size can be definitely reduced a bit. It should be easy.

**Comment by Timmy-Tesseract** [comment]
This might be a good opportunity to have a closer look at place node rendering in general. Besides the inconsistent font sizes there also seems to be a mismatch between relative importance of place and display piriority (also mentioned in [issue 3519](https://github.com/organicmaps/organicmaps/issues/3519)).

The current rendering gets even more confusing if we look at [place=quarter](https://wiki.openstreetmap.org/wiki/Tag:place%3Dquarter).  Currently this is rendered (and displayed) identical to place=neighbourhood despite it being defined as an intermediate unit between neighbourhood and suburb. 

______________________

Here an example of a place=village (orange) that consists of several place=quarter (purple) that are all further subdivided into smaller place=neighbourhood (green). The most important name, the name of the village is not displayed at all while the hierarchy between quarters and neighbourhoods is completely invisible. 

![1](https://user-images.githubusercontent.com/71176747/198511514-2c25a015-ebc6-4abb-a8c9-641b8a706f2a.png)

______________________

Another example of a place=suburb (blue) subdivided into several place=quarter (purple) and many place=neighbourhood (green). The suburb which encompasses all the other quarters and neighbourhoods should be much more recognizable. Also the quarters should have higher display priority than neighbourhood, here both that are within extent of screen are not visible while very minor neighbourhoods are displayed.

![2](https://user-images.githubusercontent.com/71176747/198513379-79ac4232-1f27-48f9-80a9-c5adeddad3d0.png)

______________________

Would it be difficult to implement strictly hierarchical display priorities and font sizes?

First display priority, largest font size: place=village/suburb
Intermediate display priority, medium font size: place=quarter
Lowest display priority, smallest font size: place=neighbourhood

**Comment by biodranik** [comment]
No, it's not difficult. Just edit the mapcss styles file accordingly.
